backend/src/test/java/codezap/template/service/facade/TemplateApplicationServiceTest.java

```diff
@@ -193,6 +244,108 @@ static Stream<Arguments> findAllBy() {
+            var category = categoryRepository.save(Category.createDefaultCategory(member));
+            for (int i = 0; i < 20; i++) {
+                var template = templateRepository.save(new Template(member, "title" + i, "description" + i, category));
+                var sourceCode = sourceCodeRepository.save(new SourceCode(template, "filename" + i, "content" + i, 1));
+                thumbnailRepository.save(new Thumbnail(template, sourceCode));
+            }
+        }
+    }
+
+    @Nested
+    @DisplayName("템플릿 목록 조회 (회원)")
+    class FindAllByWithMember {
+
+        @ParameterizedTest
+        @MethodSource
+        @DisplayName("사용자ID, 검색어, 카테고리ID, 태그ID로 템플릿 검색 성공")
+        void findAllBy(
+                Long memberId,
+                String keyword,
+                Long categoryId,
+                List<Long> tagIds,
+                Pageable pageable
+        ) {
+            // given
+            var member = memberRepository.save(MemberFixture.getFirstMember());
+            var category = categoryRepository.save(Category.createDefaultCategory(member));
+            var template1 = templateRepository.save(new Template(member, "title1", "description", category));
+            var sourceCode1 = sourceCodeRepository.save(new SourceCode(template1, "filename1", "content", 1));
+            thumbnailRepository.save(new Thumbnail(template1, sourceCode1));
+
+            // when & then
+            assertThatCode(() -> sut.findAllByWithMember(memberId, keyword, categoryId, tagIds, pageable, member))
+                    .doesNotThrowAnyException();
+        }
+
+        static Stream<Arguments> findAllBy() {
+            return Stream.of(
+                    Arguments.of(null, null, null, null, Pageable.ofSize(1)),
+                    Arguments.of(1L, null, null, null, Pageable.ofSize(1)),
+                    Arguments.of(null, "keyword", null, null, Pageable.ofSize(1)),
+                    Arguments.of(null, null, 1L, null, Pageable.ofSize(1)),
+                    Arguments.of(null, null, null, List.of(1L, 2L), Pageable.ofSize(1)),
+                    Arguments.of(null, null, null, null, Pageable.ofSize(2))
+            );
+        }
+
+        @Test
+        @DisplayName("좋아요 정보 조회 테스트")
+        void findAllByIsLikedTest() {
+            // given
+            Member loginMember = memberRepository.save(MemberFixture.getFirstMember());
+            saveDummyTemplates20();
+            List<Long> likeTemplatesIds = List.of(1L, 5L, 7L, 9L, 13L);
+            for (long likeTemplateId : likeTemplatesIds) {
+                likesRepository.save(templateRepository.fetchById(likeTemplateId).like(loginMember));
+            }
+
+            // when
+            List<FindAllTemplateItemResponse> templates =
+                    sut.findAllByWithMember(null, null, null, null,
+                                    Pageable.ofSize(1), loginMember)
+                            .templates();
```

Comment:
정렬이 뭔가 이상한 것 같....

Reply:
여기 정렬 어캐해도 이상해보여서...ㅠㅠ 
어떻게 하는 것이 그나마 예쁠까요 선생님들...

Reply:
```suggestion
            List<FindAllTemplateItemResponse> templates =
                    sut.findAllByWithMember(null, null, null, null, Pageable.ofSize(1), loginMember)
                            .templates();
```

이렇게 해도 자동정렬 다시 안되서 괜찮을 것 같아요 ~

Reply:
오호라, 단지 파라미터 정보를 함께 표시해 주느라 라인이 길어진 것 처럼 보였나보네요~